### PR TITLE
L 1197 missing element

### PR DESCRIPTION
**Pull request by vsevolod-adrianov**
Add test in ButtonTests for missing button
All right works after terminal>mvn clean complie

### CODE REVIEW

File: jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/common/ButtonTests.java [hunk 1]
```diff
@@ -9,14 +9,8 @@
...
 import static io.github.com.StaticSite.html5Page;
-import static io.github.com.pages.HtmlElementsPage.blueButton;
-import static io.github.com.pages.HtmlElementsPage.dblClickButton;
-import static io.github.com.pages.HtmlElementsPage.disabledButton;
-import static io.github.com.pages.HtmlElementsPage.disabledButtonInput;
-import static io.github.com.pages.HtmlElementsPage.ghostButton;
-import static io.github.com.pages.HtmlElementsPage.redButton;
-import static io.github.com.pages.HtmlElementsPage.rightClickButton;
-import static io.github.com.pages.HtmlElementsPage.suspendButton;
+import static io.github.com.pages.HtmlElementsPage.*;
```
Comment (codacy-review) [inline jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/common/ButtonTests.java]: Codacy Issue found: [Avoid unused imports such as 'io.github.com.pages.HtmlElementsPage'](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4699135)

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4699135&bid=15703700)

File: jdi-light/src/main/java/com/epam/jdi/light/elements/common/UIElement.java [hunk 2]
```diff
@@ -669,6 +673,11 @@ protected boolean displayed() {
         return false;
     }
 
+    protected boolean missing() {
+        List<WebElement> result = getAllElements();
+        return result.size() == 0;
```
Comment (Inov92) [inline jdi-light/src/main/java/com/epam/jdi/light/elements/common/UIElement.java]: Improve code clearness by using List method isEmpty

**Comment by mf-code** [comment]
[4194] - https://transfer.sh/113x9G/jdi-performance.tar.gz

**Comment by mf-code** [comment]
[4194] - https://transfer.sh/xFnQb/jdi-light-html-tests.tar.gz

**Comment by mf-code** [comment]
[4194] - https://transfer.sh/12lbWf/jdi-light-examples.tar.gz

**Comment by mf-code** [comment]
[4194] - Allure report on Netlify: https://5dfa3ccd29a447e06ca6bc57--musing-swartz-6fc060.netlify.com

**Comment by mf-code** [comment]
[4198] - https://transfer.sh/1DWjB/jdi-performance.tar.gz

**Comment by mf-code** [comment]
[4198] - https://transfer.sh/bYoOT/jdi-light-html-tests.tar.gz

**Comment by mf-code** [comment]
[4198] - https://transfer.sh/Kw9Sf/jdi-light-examples.tar.gz

**Comment by mf-code** [comment]
[4198] - Allure report on Netlify: https://5dfa456811853b3b0d46c86a--musing-swartz-6fc060.netlify.com

**Comment by mf-code** [comment]
[4202] - https://transfer.sh/72bvI/jdi-performance.tar.gz

**Comment by mf-code** [comment]
[4202] - https://transfer.sh/HZ6hM/jdi-light-html-tests.tar.gz

**Comment by mf-code** [comment]
[4205] - https://transfer.sh/yGrdA/jdi-light-html-tests.tar.gz

**Comment by mf-code** [comment]
[4205] - https://transfer.sh/BGBAu/jdi-performance.tar.gz

**Comment by mf-code** [comment]
[4205] - https://transfer.sh/jMyBz/jdi-light-examples.tar.gz

**Comment by mf-code** [comment]
[4205] - Allure report on Netlify: https://5dfa5aab56a2cec6870fefd0--musing-swartz-6fc060.netlify.com

**Pull request closed without merging**